step-6/README.md

```diff
@@ -1,7 +1,9 @@
 # Step 6: Complete the game
+
 You've come a long way!
 
-If you were able to get your `matchCards()` method working at the end of [Step 5]() on your own, then you should the `matchCards()` function code, with all the component variables and methods prefixed with `this.` in your `GameBoard.vue`component:
+If you were able to get your `matchCards()` method working at the end of [Step 5]() on your own, then you should the `matchCards()` function code, with all the component variables and methods prefixed with `this.` in your `GameBoard.vue` component:
```

Comment:
Would you mind adding a link to the 5th step here? Apparently I missed that this link was not functional - it should be! Thanks 😃

Reply:
Added a relative path to step-5